letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -244,15 +244,29 @@ DeprecationBootstrap() {
   fi
 }
 
-
+# Sets LE_PYTHON to Python version string and PYVER to the first two
+# digits of the python version
 DeterminePythonVersion() {
-  for LE_PYTHON in "$LE_PYTHON" python2.7 python27 python2 python; do
-    # Break (while keeping the LE_PYTHON value) if found.
-    $EXISTS "$LE_PYTHON" > /dev/null && break
-  done
+  # Arguments: "NOCRASH" if we shouldn't crash if we don't find a good python
+  if [ -n "$USE_PYTHON_3" ]; then
```

Comment:
While this is technically correct, we might dodge a future surprise if we use `[ -n "$USE_PYTHON_3" != "1" ]` instead, so somebody doesn't come along later, set USE_PYTHON_3=0, and expect the obvious thing to happen.